config/peerpods/peerpodssecret.yaml

```diff
@@ -10,4 +10,4 @@ stringData:
   #LIBVIRT_NET: "default
   #LIBVIRT_POOL: "default"
   #REDHAT_OFFLINE_TOKEN: "" #Required to download rhel base image : Download token from https://access.redhat.com/management/api
-  
\ No newline at end of file
+  #HKD_CRT: "" #Download the certificate from https://www.ibm.com/support/resourcelink/api/content/public/host-key-documents.html
```

Comment:
How about naming it to `HOST_CERTS`  or `HOST_KEY_CERTS` ?
It's ok to be little descriptive in the config param imho.